apps/experiments/models.py

```diff
@@ -453,6 +453,10 @@ class Participant(BaseTeamModel):
     user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, null=True, blank=True)
     platform = models.CharField(max_length=32)
 
+    class Meta:
+        ordering = ["platform", "identifier"]
+        unique_together = [("team", "platform", "identifier")]
```

Comment:
Just confirming we don't also want to index participant on the platform field?

Reply:
Do you mean a separate index for `platform`? I don't think that's necessary.